source/_patterns/04-pages/article--research~with-structured-abstract.json

```diff
@@ -590,11 +590,11 @@
             },
             {
               "primary": {
-                "name": "Executable DAR",
+                "name": "Executable version",
                 "url": "#darLink"
```

Comment:
nit pick: let's change this to #eraLink as it will no longer be DAR files.

Reply:
Optionally, you could also just remove the links to the `Executable version` from this example article so that we just have one example? No preference.